xla/stream_executor/rocm/rocm_blas.cc

```diff
@@ -363,6 +363,12 @@ absl::Status ROCMBlas::DoBlasInternalImpl(FuncT rocblas_func, Stream *stream,
                  << ": " << ToString(ret);
     }
   }
+#if TF_ROCM_VERSION >= 60000
+  if (auto *workspace = GetWorkspace();
+      workspace && workspace->opaque() && workspace->size() > 0) {
```

Comment:
The very first part of the && check is there to just check whether pointer is pointing to non-null address, right?

Can we make it explicit, like `workspace != nullptr`?

Reply:
yes, I have fixed it